place_tracker/lib/place_map.dart

```diff
@@ -93,7 +93,7 @@ class _PlaceMapState extends State<PlaceMap> {
   @override
   Widget build(BuildContext context) {
     _watchMapConfigurationChanges();
-    var state = Provider.of<AppState>(context);
+    var state = Provider.of<AppState>(context, listen: true);
```

Comment:
It is necessary to listen to state changes here, the `listen: true` was missing.